sdcm/utils/sstable/load_utils.py

```diff
@@ -51,26 +52,41 @@ def distribute_test_files_to_cluster_nodes(cls, nodes, test_data: List[TestDataI
         return map_files_to_node
 
     @staticmethod
-    def upload_sstables(node, test_data: TestDataInventory, keyspace_name: str = 'keyspace1'):
+    def upload_sstables(node, test_data: TestDataInventory, keyspace_name: str = 'keyspace1',
+                        create_schema: bool = False, **kwargs):
         key_store = KeyStore()
         creds = key_store.get_scylladb_upload_credentials()
         # Download the sstable files from S3
         remote_get_file(node.remoter, test_data.sstable_url, test_data.sstable_file,
                         hash_expected=test_data.sstable_md5, retries=2,
                         user_agent=creds['user_agent'])
-        result = node.remoter.sudo(f"ls -t /var/lib/scylla/data/{keyspace_name}/")
-        upload_dir = result.stdout.split()[0]
-        if node.is_docker():
-            node.remoter.run(f'tar xvfz {test_data.sstable_file} -C /'
-                             f'var/lib/scylla/data/{keyspace_name}/{upload_dir}/upload/')
-        else:
-            node.remoter.sudo(
-                f'tar xvfz {test_data.sstable_file} -C /var/lib/scylla/data/{keyspace_name}/{upload_dir}/upload/',
-                user='scylla')
-
-        # Scylla Enterprise 2019.1 doesn't support to load schema.cql and manifest.json, let's remove them
-        node.remoter.sudo(f'rm -f /var/lib/scylla/data/{keyspace_name}/{upload_dir}/upload/schema.cql')
-        node.remoter.sudo(f'rm -f /var/lib/scylla/data/{keyspace_name}/{upload_dir}/upload/manifest.json')
+
+        tmp_folder = "/tmp/snapshot/"
+        with RemoteTemporaryFolder(node=node, folder_name=tmp_folder):
+
+            if node.is_docker():
+                node.remoter.run(f'tar xvfz {test_data.sstable_file} -C {tmp_folder}')
+            else:
+                node.remoter.sudo(f'tar xvfz {test_data.sstable_file} -C {tmp_folder}', user='scylla')
+
+            if create_schema:
+                SstableLoadUtils.create_keyspace(node=node,
+                                                 replication_factor=kwargs["replication_factor"],
+                                                 drop_ks_if_exists=kwargs["drop_ks_if_exists"]
```

Comment:
Place at the end either comma or ending parentheses.

Reply:
ending with parentheses